pkg/volume/iscsi/iscsi_util.go

```diff
@@ -167,7 +171,11 @@ func (util *ISCSIUtil) AttachDisk(b iscsiDiskMounter) error {
 
 	iscsiTransport = extractTransportname(string(out))
 
-	bkpPortal := b.portals
+	bkpPortal := removeDuplicate(b.portals)
```

Comment:
Why we need `removeDuplicate` in attachdisk, can you please give some details on why we need this ?